admin/onpage/class-onpage.php

```diff
@@ -33,16 +33,16 @@ public function __construct() {
 	 * @return void
 	 */
 	public function register_hooks() {
-		if ( ! $this->is_active() ) {
+		// Adds admin notice if necessary.
+		add_filter( 'admin_init', array( $this, 'show_notice' ) );
+
+		if ( ! self::is_active() ) {
```

Comment:
We also set a cronjob. Do we remove this when Ryte is not active?

Reply:
We only add a schedule, but that schedule is not being added if we exit early.

Reply:
You are correct, will implement.